cleaner.go

```diff
@@ -232,6 +248,49 @@ func (c *Cleaner) setLaunchTemplateUsed() error {
 	return nil
 }
 
+// If the passed imageAlias is an alias, It will return the resolved imageId.
+// Otherwise, it will return the original imageId.
+// See https://docs.aws.amazon.com/autoscaling/ec2/userguide/using-systems-manager-parameters.html to understand this use case.
+func (c *Cleaner) resolveImageAlias(imageAlias string, launchTemplateId *string, launchTemplateVersion *int64) (string, error) {
+	// If we need to resolve aliases. We need to perform an additional DescribeLaunchTemplateVersions
+	// for each version we want to resolve. Because when calling the DescribeLaunchTemplateVersions operation: Resource aliasing (resolveAlias)
+	// is only supported when doing single version describe
```

Comment:
Is there any official documentation that explains this limitation? I'm wondering if this is really the intended behavior.
If true, AWS should add the `ResolveAlias` ​​option to the API to get a single template version...

Reply:
It's not documented. However when I tried passing the `ResolveAlias` set to true on the initial call to DescribeLaunchTemplateVersions. I got the error: `Resource aliasing (resolveAlias) is only supported when doing single version describe`. Therefore, I assume this is the intended behaviour.